sdk/python/feast/client.py

```diff
@@ -282,6 +297,16 @@ def version(self):
 
         return result
 
+    def set_tracer(self, tracer: Tracer):
+        """
+        Set the OpenTracing Tracer instance used to trace the gRPC calls to Feast Core
```

Comment:
Do we need to trace to Core as well? Is that even implemented?

Reply:
It is not implemented on server side. So, if we include the interceptor here, it will only capture the latency as observed by the client side (and ingress gateway, if set up)

The reason i include Feast core, is because Feast client is shared by both serving and core, and it seems a little unnatural that setting a tracer affects only serving but not core. Either way, I can rename set_tracer to set_serving_tracer, and remove the interceptor for Feast core, if we don't want to trace the call to Feast Core?